### PR TITLE
Smth fuzz audio updates

### PR DESCRIPTION
**Pull request by NoSloppy**
Added boot.wav.
Added endlock made from lock + SmthGrey clash04.
hum.wav Normalized to -5db.
High passed hum @100hz - 48db slope.
out.wav Normailized to 0dB and fade out full.
Edited lock to loop cleanly and have less monophonic-like hum mixed in,
and additional crackling mixed in.

### CODE REVIEW

**Comment by NoSloppy** [comment]
Updated- no highpass

**Comment by profezzorn** [comment]
needs readme.txt updates

File: SmthFuzz/license.txt [hunk 1]
```diff
@@ -1 +1,11 @@
-These files are free to use, modify, and redistribute without attribution or any other restrictions.
\ No newline at end of file
+These files are free to use, modify, and redistribute without attribution or any other restrictions.
+
+Modifications by Brian Conner 2022:
+Detailed changes can be found in this font's ReadMe.txt.
+Any additional sounds sourced from creative-commons licensed freesound.org.
```
Comment (profezzorn) [inline SmthFuzz/license.txt]: specify files

Comment (NoSloppy) [reply SmthFuzz/license.txt]: Done.

**Pull request merged**